### ISSUE TITLE
React native android duplicate data

### ISSUE THREAD

**Issue by tonigue** [rajdeepak27/react-native-shield-fraud-plugin#4]
hello i just tried to implement react native shield. on ios release or debug mode the data sent to the shield dashboard corresponds to only one. but when trying on android the data reads duplicate with time up to the same second. please for an explanation. here is the code i used:

```
    "react": "17.0.2",
    "react-native": "0.68.3",
    "react-native-shield-fraud-plugin": "^1.0.8",
```
in app.js

```
async initializeShield() {
    await ShieldFraud.initShield({
      siteID: Config.SHIELD_SITE_ID,
      secretKey: Config.SHIELD_SECRET_KEY,
      blockedDialog: {
        title: "Blocked Dialog Title",
        body: "Blocked Dialog Body",
      }
    })
  }

componentDidMount() {
this.initializeShield()
}

```

in other screen:

```
 ShieldFraud.isSDKready(async (isReady) => {
            if (isReady) {
              console.log("SDK is ready for ShieldFraud");

              ShieldFraud.sendAttributes("Login", { userId: dtUser?.uid });
              ShieldFraud.getLatestDeviceResult()
                .then((result) => {
                  // Handle success with the result object
                  console.log(
                    "ShieldFraud Received latest device result:",
                    result
                  );
                })
                .catch((error) => {
                  // Handle error with the error object
                  console.log("Error retrieving device result:", error);
                });
            } else {
              console.log("SDK is not ready for ShieldFraud");
            }
          })
```

the result:
![Screenshot 2024-06-10 at 17 06 19](https://github.com/shield-ai-technology/react-native-shield-fraud-plugin/assets/134369203/c1e7e184-a0a2-4c93-9d07-80f8422e4bcd)

**Comment by yemyatthu1990** [comment]
@rajdeepak27 I think the issue is because of this line. 
https://github.com/shield-ai-technology/react-native-shield-fraud-plugin/blob/a3fa244ba00c996dbef968b721578edf8d371e0a/android/src/main/java/com/shieldfraudplugin/ShieldFraudPluginModule.java#L183

We're calling getLatestDeviceResult twice just to check whether it's null or not.

**Comment by rajdeepak27** [comment]
Thanks @tonigue, The Issue is fixed in 1.0.9 release here - https://github.com/shield-ai-technology/react-native-shield-fraud-plugin/commit/70e13f57d3ff8b25189b2be7d151320c79283af1
